include/IMeshBuffer.h

```diff
@@ -161,7 +162,7 @@ GL_INVALID_OPERATION is generated by glVertexAttribPointer if size is GL_BGRA an
 
 
 	template <class T>
-	class IMeshDataFormatDesc : public virtual IReferenceCounted
+	class IMeshDataFormatDesc : public virtual IReferenceCounted, public core::BlobSerializable<IMeshDataFormatDesc<core::ICPUBuffer> >

```

Comment:
this makes the entire template class serializable, not just ICPUMeshDataFormatDesc (which is a typedef for IMeshDataFormatDesc<ICPUBuffer>)

will have to convert the ICPUMeshDataFormatDesc typedef into a class inheriting from IMeshDataFormatDesc<ICPUBuffer> and BlobSerializable

Reply:
Yea, I know it's a todo, got it in mind